resources/views/auth/login.blade.php

```diff
@@ -1,6 +1,6 @@
-@extends('layouts.app')
+@extends('layouts.prelogin')
 
-@section('content')
+@section('article_list')
```

Comment:
これもいきなり`snake_case`で書くのやめた方がいいよ。
dot notationで書くので統一すべき。

```suggestion
@section('article.list')
```

Reply:
修正済。